service/ztests/curl-default-poolkey.yaml

```diff
@@ -0,0 +1,30 @@
+script: |
+  source service.sh
+  curl -X POST -d '{"name":"test"}' $ZED_LAKE/pool > pool.json
+  curl -X POST \
+    -H "Accept: application/json" \
+    -d '{"query":"from :pools"}' \
+    $ZED_LAKE/query | zq -Z "yield this[0] | id:=0,ts:=0" -
```

Comment:
Instead of adding this test, just remove `defaultSortKey` from service/handlers_test.go so the tests in there get the actual default.